dwave_networkx/generators/pegasus.py

```diff
@@ -1178,3 +1186,89 @@ def source_to_inner(q):
     for offset in offset_list:
         yield make_mapping(source_to_inner, nice_to_target, offset)
 
+
+def pegasus_torus(m, node_list=None, edge_list=None, 
+                  offset_lists=None, offsets_index=None):
+    """
+    Creates a Pegasus graph modified to allow for periodic boundary conditions and translational invariance.
+
+    Parameters
+    ----------
+    m : int
+        Size parameter for the Pegasus lattice.
+        Connectivity of all nodes is :math:`13+min(m-1,2)`
+    node_list : iterable, optional (default None)
+        Iterable of nodes in the graph. If None, calculated from `m`.
+        Note that this list is used to remove nodes, so any nodes specified
+        not in ``range(24 * m * (m-1))`` are not added.
+    edge_list : iterable, optional (default None)
+        Iterable of edges in the graph. If None, edges are generated as
+        described below. The nodes in each edge must be integer-labeled in
+        ``range(24 * m * (m-1))``.
+    offset_lists : pair of lists, optional (default None)
+        Directly controls the offsets. Each list in the pair must have length 12
+        and contain even ints.  If `offset_lists` is not None, the `offsets_index`
+        parameter must be None.
+    offsets_index : int, optional (default None)
+        A number between 0 and 7, inclusive, that selects a preconfigured
+        set of topological parameters. If both the `offsets_index` and
+        `offset_lists` parameters are None, the `offsets_index` parameters is set
+        to zero. At least one of these two parameters must be None.
+
+    Returns
+    -------
+    G : NetworkX Graph
+        A Pegasus torus for size parameter `m` using the coordinate labeling system.
+
+
+    A pegasus torus is a generalization of the standard pegaus graph
+    whereby bulk connectivity properties are maintained, but the boundary
+    condition is modified to enforce an additional translational 
+    invariance symmetry [RH]_. Local connectivity in the pegasus torus
+    is identical to connectivity for pegasus graph nodes away from the boundary.
+    A tile consists of 24 nodes, and the torus has m-1 by m-1 tiles. 
```

Comment:
```suggestion
    A tile consists of 24 nodes, and the torus has :math:`m-1` by :math:`m-1` tiles. 
```

Note that we don't really define "tile", the [Pegasus description](https://docs.dwavesys.com/docs/latest/c_gs_4.html#pegasus-couplers) talks about "A Pegasus unit **cell** contains twenty-four qubits"